.github/workflows/go.yml

```diff
@@ -79,6 +79,26 @@ jobs:
     - name: Test
       run: make test
 
+    # Launcher should always print its version info when called with `version`, which includes the golang version;
+    # we confirm that it contains that info here.
+    - name: Test build - macOS and ubuntu
+      if: ${{ contains(matrix.os, 'macos') || contains(matrix.os, 'ubuntu') }}
+      run: |
+        if [[ $(./build/launcher version) != *go${{env.GOVER}}* ]]; then
```

Comment:
Could check for the string `launcher - version` which is probably what update does. But I'd approve this

Reply:
I like that better, it's more readable without the environment variable in there 🙂 updated